src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig

```diff
@@ -160,8 +167,7 @@
                 <input type="button" class="btn btn-outline-secondary" name="products_filter_position_asc" value="{{ "Reorder"|trans({}, 'Admin.Actions') }}" onclick="productOrderPrioritiesTable();" />
```

Comment:
Actually, the wording _Reorder_ is confusing for translators - some get it as ordering something again while it should be understood in the context of changing position... perhaps we should think of a better formulation for future versions? Now it is too late because of string freeze but let's keep it in mind! (w/ @marionf)

Reply:
Hey again, the Crowdin community suggested 'Sort order' instead of 'Reorder' to avoid confusion in the back office, what do you think about that?